src/sys/fuchsia/net.rs

```diff
@@ -245,12 +245,10 @@ impl TcpListener {
         })
     }
 
-    #[allow(deprecated)]
     pub fn set_only_v6(&self, only_v6: bool) -> io::Result<()> {
```

Comment:
This fn and `only_v6` probably can be deleted now?

Reply:
Removed. What about the other `only_v6` and `set_only_v6` functions in the crate? Should they all be removed?